### PR TITLE
2nd iCBT: Minor fixes

### PR DESCRIPTION
**Pull request by Sunaries**
@imcgames 

GitHub Logo
1. Can you please change the 'Hit' to 'Hot'. It can confuse players that this is a hitting skill. Because the class advancement window does say 'Hot' for advancement classes.
2. Why was the [Translation Bug Report category](https://forum.treeofsavior.com/c/beta-test-bug-report/translatio) on the forums disabled? Please enable it and tell users to post their discoveries there, so that it is easier for us to navigate and fix translation-wise errors.

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -11122,7 +11122,7 @@ ETC_20150414_011123	{memo X}The region where Parties are most active
 ETC_20150414_011124	{memo X}The greatest enhancement item(under development)
 ETC_20150414_011125	{memo X}The person who obtained the amount of silvers
 ETC_20150414_011126	{memo X}Male, Female Proportion(under development)
-ETC_20150414_011127	Talk to the summoned guardian of Zachariel to activate the altar!
+ETC_20150414_011127	Talk to the summoned protector of Zachariel to activate the altar!
```
Comment (vtange) [inline ETC.tsv]: I thought we standardized Protectors to Guardians in Mausoleum

File: ETC.tsv [hunk 2]
```diff
@@ -11247,7 +11247,7 @@ ETC_20150428_011248	$Defeat Helgasercle, who is occupying the Mage Tower!
 ETC_20150428_011249	$Defeat Bearkaras for Simon Shaw!
 ETC_20150428_011250	The Stone Whale guarding Hauberk has woken up. {nl}Defeat the Stone Whale!
 ETC_20150428_011251	Aaaagh!
-ETC_20150428_011252	The Devout Tree came to life again as the Spring of Silence was purified!{nl}But an evil force is felt from the Peddler's Camp!
+ETC_20150428_011252	The Devout Tree came to life again as the Spring of Silence was purified!{nl}But an evil energy is felt from the Peddler's Camp!
```
Comment (vtange) [inline ETC.tsv]: Devout Tree -> Tree of Faith
it's a sub-area name

File: ETC.tsv [hunk 3]
```diff
@@ -12733,7 +12733,7 @@ ETC_20150714_012734	You didn't find any peculiar information
 ETC_20150714_012735	It will be dangerous if you go near
 ETC_20150714_012736	The Eyes of the Great King is whispering to you that the right person has come
 ETC_20150714_012737	You can only use it in the Lot 2 of the Crystal Mine
-ETC_20150714_012738	Defeat the monsters that protect the demon summon field first!
+ETC_20150714_012738	Defeat the monsters first that protect the demon summoning circle!
```
Comment (vtange) [inline ETC.tsv]: Defeat the monsters protecting the demon summoning circle first!

File: ETC.tsv [hunk 4]
```diff
@@ -12752,22 +12752,22 @@ ETC_20150714_012753	$The demon soul has been eradicated.
...
 ETC_20150714_012758	$You need to defeat the demons near the Barrier to charge it.
-ETC_20150714_012759	$You've charged the Barrier of Tree Guard Post a bit
+ETC_20150714_012759	$You've charged the Tree Guard Post Barrier a bit
 ETC_20150714_012760	$You scribbled on the demon summoning circle
 ETC_20150714_012761	You don't have enough Vicious Essences
 ETC_20150714_012762	You have obtained a Purified Essence
 ETC_20150714_012763	You have obtained a Vicious Essence
 ETC_20150714_012764	You have obtained enough Purified Essences
-ETC_20150714_012765	You have drunk Romuva Sacred Water.{nl}You became invinsible to demons temporarily!
+ETC_20150714_012765	You have drunk the Romuva Holy Water.{nl}You temporarily became invinsible to the demons!
```
Comment (vtange) [inline ETC.tsv]: invinsible -> invisible

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -1391,7 +1391,7 @@ QUEST_LV_0100_20150317_001386	Powerless Security Guard
 QUEST_LV_0100_20150317_001387	I was attacked by Hogmas at Traceless Ruins.{nl}Fortunately, I was able to survive, but I need to find that box.
 QUEST_LV_0100_20150317_001388	Thanks for caring.{nl}Be careful of those Hogmas.
 QUEST_LV_0100_20150317_001389	They probably haven't gone that far.{nl}I hope the necklace is safe even if others are not.
-QUEST_LV_0100_20150317_001390	There was nothing at the box?{nl}Oh my Goddess, that's not good.
+QUEST_LV_0100_20150317_001390	There was nothing in the chest?{nl}Oh my Goddess, that's not good.
```
Comment (vtange) [inline QUEST_LV_0100.tsv]: that can't be good.

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Comment by imcgames** [comment]
Thank you @Sunaries 
I understand why we need to change "Hit" to "hot". But, in order to change it, i think we need some times. I will check it with our design team.

You mean this category on the Forum? it's already enable.
https://forum.treeofsavior.com/c/beta-test-bug-report/translation

**Comment by imcgames** [comment]
:+1:

**Comment by Sunaries** [comment]
@imcgames >You mean this category on the Forum? it's already enable.

No, it isn't. :cry:  See image:

GitHub Logo

Users cannot create new topic.

**Comment by imcgames** [comment]
Thank you @Sunaries 
I'm going to check it again.

**Comment by Sunaries** [comment]
@imcgames 

Thank you, my hero(ine) :yum:

**Pull request merged**